packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html

```diff
@@ -80,17 +80,25 @@
           </li>
         </ul>
       </oui-tile>
-      <p class="mt-4"
-         data-translate="ovhManagerSupport_new_helpfull">
-      </p>
-      <oui-button data-variant="secondary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_no' | translate }}"
-                  data-on-click="$ctrl.submitForm(false)">
-      </oui-button>
-      <oui-button data-variant="primary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_yes' | translate }}"
-                  data-on-click="$ctrl.submitForm(true)">
-      </oui-button>
+      <div data-ng-if="$ctrl.guides.length || $ctrl.tips.length">
+        <p class="mt-4"
```

Comment:
Indentation size isn't correct. Should be 4 spaces for `*.html` files.

ref: https://github.com/ovh-ux/manager/blob/master/.editorconfig#L22

Reply:
i indented all the templates in new-ticket parts with 2 spaces ; i'll do another PR to fix all the indentation